libbpf-tools/biolatency.c

```diff
@@ -40,20 +40,20 @@ const char *argp_program_bug_address = "<bpf@vger.kernel.org>";
 const char argp_program_doc[] =
 "Summarize block device I/O latency as a histogram.\n"
 "\n"
-"USAGE: biolatency [--help] [-T] [-m] [-Q] [-D] [-F] [-d] [interval] [count]\n"
+"USAGE: biolatency [--help] [-T] [-M] [-Q] [-D] [-F] [-d DISK] [interval] [count]\n"
 "\n"
 "EXAMPLES:\n"
 "    biolatency              # summarize block I/O latency as a histogram\n"
 "    biolatency 1 10         # print 1 second summaries, 10 times\n"
-"    biolatency -mT 1        # 1s summaries, milliseconds, and timestamps\n"
+"    biolatency -MT 1        # 1s summaries, milliseconds, and timestamps\n"
```

Comment:
biolatency.py is using `-m`, why sudden and diverging change?

Reply:
> There are a bunch more cases of arbitrary changing the case of existing command line arguments with no justification. Please provide rationale for the changes in your commit messages, not just the "what".

Oh, my motivation is I think maybe it's easier to remember. Lowercase parameters need parameter values ​​(except -v), and uppercase ones belong to the switch class, so you don't need to add parameter values.

> Also, while I have your attention. Can you please check [#3224 (comment)](https://github.com/iovisor/bcc/issues/3224#issuecomment-759909082)? Would you be able to add that check? Thanks!

Oh sorry, I haven't checked the email from iovisor/bcc recently, I missed it. I will look at it later （Maybe weekend）.

Reply:
I haven't heard about such naming convention, tbh. I think the harm of changing existing arguments and, especially, diverging from .py tool, is much bigger. So let's not do such renamings unnecessarily.

And thanks for checking the other PR.

Reply:
Got it,  I will be consistent with .py tool. Thanks!